stack-graphs/Cargo.toml

```diff
@@ -32,7 +33,7 @@ enumset = "1.1"
 fxhash = "0.2"
 itertools = "0.10"
 libc = "0.2"
-lsp-positions = { version = "0.3", path = "../lsp-positions" }
+lsp-positions = { version = "0.3", path = "../lsp-positions", default-features=false }
```

Comment:
Disabling the `default-features` shouldn't be necessary anymore with the change above.